src/site/content/en/blog/fugu-status/index.md

```diff
@@ -0,0 +1,263 @@
+        </td>
+        <td>
+          Your PWA might cache articles and media files, but how will your users
+          know that your pages work while offline? The Content Indexing API is
+          one answer to this question currently in an origin trial. Once the
+          index is populated with content from your PWA, as well as any other
+          installed PWAs, it will show up in dedicated areas of supported
+          browsers.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <a href="/native-file-system/">
+            Native File System API
+          </a>
+        </td>
+        <td>
+          The Native File System API (formerly known as the Writable Files API)
+          enables developers to build powerful web apps that interact with files
+          on the users local device, like IDEs, photo and video editors, text
+          editors, and more. After a user grants a web app access, this API
+          allows web apps to read or save changes directly to files and folders
+          on the users device.
+        </td>
+      </tr>
+      <tr>
+        <td>Notification Triggers</td>
+        <td>
+          Notification Triggers let you schedule notifications in advance, so
+          that the operating system will deliver the notification at the right
+          time - even if there is no network connectivity, or the device is in
+          battery saver mode.
+        </td>
+      </tr>
+      <tr>
+        <td>Serial API</td>
+        <td>
+          The Web Serial API provides a way for websites to read from and
+          write to a serial device with scripts. The API bridges the web and
+          the physical world by allowing websites to communicate with serial
+          devices, such as microcontrollers and 3D printers.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <a href="/sms-receiver-api-announcement/">SMS Receiver API</a>
+        </td>
+        <td>
+          Finding, memorizing, and typing OTPs sent via SMS is cumbersome.
+          The SMS Receiver API simplifies the OTP workflow for users.
+        </td>
+      </tr>
+      <tr>
+        <td>
+          <a href="/wakelock/">Wake Lock API</a>
+        </td>
+        <td>
+          To avoid draining the battery, most devices will quickly fall asleep
+          when left idle. While this is fine for most of the time, there are
+          some applications that need to keep the screen or the device awake in
+          order to complete some work. The Wake Lock API provides a way to
+          prevent the device from dimming or locking the screen or prevent
+          the device from going to sleep when an application needs to keep
+          running.
+        </td>
+      </tr>
+    </tbody>
+  </table>
+</div>
+
+## Capabilities available in stable {: #in-stable }
+
+The following APIs have graduated from origin trial and are available in the
+latest version of Chrome, and in many cases other Chromium based browsers.
```

Comment:
(Aside: We should list the other browsers' availability.)

Reply:
MDN or CanIUse? What would you suggest for a place to link. Ideally I don't want to have to test this myself and update it every release.